src/lib.rs

```diff
@@ -3,19 +3,19 @@ Compiletime string literal obfuscation.
 !*/
 
 #![no_std]
-#![feature(fixed_size_array)]
 
 // Reexport these because reasons...
 #[doc(hidden)]
 pub use obfstr_impl::*;
+pub use obfstr_impl::wide;
 
 #[cfg(feature = "rand")]
 pub use cfgd::*;
 #[cfg(feature = "rand")]
 mod cfgd {
 
 use core::{char, fmt, mem, ops, ptr, slice, str};
-use core::array::FixedSizeArray;
+pub use obfstr_impl::random;
```

Comment:
Why export those publicly? I think it's better to keep them as private impl details.

Reply:
The interface is identical between what is defined as a proc-macro and the declarative macro. There's no need to wrap the proc-macro again. The only reason it was done was because function-like proc-macros weren't stabilized but they are now.

If necessary it can be reverted without breaking changes as the whole proc-macro crate is an implementation detail anyway.

Reply:
No I mean, this pub use will appear in the rustdoc, e.g. 
![image](https://user-images.githubusercontent.com/1069318/97109634-998d8a80-16d4-11eb-9813-ed6a6e4d2df3.png)

Those should be `doc(hidden)`, as they're private impl details that we don't expect users to be using.

Reply:
Wait, nvm this, I just realized 0.1.1 already exports random and wide publicly. I should have double-checked.

Reply:
Yeah they're part of this crate's public interface. I don't like having a lot of tiny crates for such functionality and I need these utilities anyway so I might as well export them here. In fact there's a bunch of other users that depend on obfstr solely for its wide macro.